### ISSUE TITLE
Port 5000 in Use by uPNP

### ISSUE THREAD

**Issue by BzowK** [wolandmaster/luci-app-rtorrent#22]
Hey Guys - 

I'm running LEDE 17.01.4 on a Linksys WRT1900ACS and today thought I'd try our rtorrent on it and manage it via LUCI.  I installed and configured everything as listed here and after my issue also tried alternate init.d scripts like the one listed for Issue #7.  Unfortunately, I cannot get it to work as when I try to view anything related from LUCI, I eventually get the "Bad Gateway" error.

Seemingly, the issue is that I have uPNP enabled thus causing a port conflict.  When executing "netstat -pantu | grep 5000", I get the below:
```
tcp        0      0 :::5000                 :::*                    LISTEN      4192/miniupnpd
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.125:64043 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.125:64040 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.166:59718 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.125:64034 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.116:50237 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.116:50196 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.125:64052 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.25:43961 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.125:64035 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.125:64037 TIME_WAIT   -
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.166:59724 TIME_WAIT   -
tcp        0      1 ::1:49244               ::1:5000                SYN_SENT    6134/lua
tcp        0      0 ::ffff:192.168.0.1:5000 ::ffff:192.168.0.125:64038 TIME_WAIT   -
```
_My router is at 192.168.0.1 and the IPs referenced above are all workstations or VMs_

Also, if trying to manually launch rtorrent from SSH, I get:
`rtorrent: Error in option file: ~/.rtorrent.rc:9: Could not prepare socket for listening: Address in use.`

I'd imagine that changing the value of "scgi_port" within /root/.rtorrent.rc to a different ip:port would take care of having it listen on a different port, but is luci-app-rtorrent hardcoded to port 5000 or can that easily be changed to match it as well?  Yes, I could also disabled uPNP as I have all ports manually forwarded already.  I know there was a reason I enabled it, but don't remember what it was exactly :)

Suggestions appreciated - Thanks!

**Comment by wolandmaster** [comment]
The port in the latest released luci-app-rtorrent is hardcoded, however, it is very easy to change. You can do it in /usr/lib/lua/rtorrent.lua file.
(it will be configurable by LuCI gui in the next version)

**Comment by wolandmaster** [comment]
Duplicate of #14.

**Comment by wolandmaster** [comment]
Fixed in [0.2.2](https://github.com/wolandmaster/luci-app-rtorrent/releases/tag/0.2.2)
